events/2025-03-10-hts-workshop-freiburg.md

```diff
@@ -0,0 +1,117 @@
+---
+layout: event
+title: "Workshop on high-throughput sequencing data analysis with Galaxy"
+
+# external: "https://galaxyproject.org/events/2025-03-10-galaxy-workshop-freiburg/"
+
+description: |
+    This course introduces scientists to the data analysis platform Galaxy. The course is a beginner course; no programming skills are required.
+
+date_start: 2025-03-10
+date_end: 2025-03-14
+
+cost: free
+audience: Scientists with no or little Galaxy experience who want to analyse sequencing data.
+contact_email: schneidd@informatik.uni-freiburg.de
+# async: false
+mode: onsite
+
+registration:
+  link: https://forms.gle/cC9ynBrypEWwNA9u8
+  deadline: 2025-02-14
+
+contributions:
+  organisers: [Sch-Da, teresa-m]
+  instructors: [pavanvidem, Nilchia, teresa-m, wm75, EngyNasr, paulzierep]
+  funding: [eurosciencegateway, deNBI, nfdi4plants]
+
+location:
+  geo:
+    lat: 47.993460
+    lon: 7.845110
```

Comment:
(fyi you don't have to fill this in actually! the automation will take care of it for you.)

Reply:
Can I just add the street?

Reply:
yeah if you just add the street address that's fine

Reply:
So I could add just geo with nothing else like this:
```
location
  geo:
  name: University Freiburg
  ```

Reply:
you don't need to add the geo key at all